Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/policyengine/adapter/pap/AdapterPAPImpl.java

```diff
@@ -62,13 +65,12 @@ public Document getPolicyDocumentByPatientId(String patientId) {
             DocumentService service = new DocumentService();
             List<Document> docs = service.documentQuery(params);
             int docsSize = 0;
-            if (docs != null) {
+            if (!CollectionUtils.isEmpty(docs)){
```

Comment:
you can use - CollectionUtils.isNotEmpty(docs) -this will be more clearer and easy to read